build/repo.targets

```diff
@@ -0,0 +1,17 @@
+﻿<Project>
+  <Import Project="VSIX.targets" />
+  <ItemGroup>
+    <Solutions Update="$(RepositoryRoot)Blazor.sln">
+      <!-- the 'DebugNoVSIX' and 'ReleaseNoVSIX' configurations exclude the VSIX project, which doesn't build with Microsoft.NET.Sdk yet. -->
+      <AdditionalProperties>Configuration=$(Configuration)NoVSIX</AdditionalProperties>
+    </Solutions>
+  </ItemGroup>
+
+  <!-- 
+    Get Anglesharp built right away. This isn't needed for correctness, but helps avoid concurrency problems.
+  -->
+  <Target Name="BuildAngleSharp" BeforeTargets="BuildSolutions">
```

Comment:
This was one issue that took a lot of time. Our command line builds haven't been too reliable because of the way we deal with anglesharp. I added a few mitigations and it seems pretty stable now.